crates/workspace/src/workspace.rs

```diff
@@ -2581,6 +2584,9 @@ impl Workspace {
                 }
             }
             pane::Event::Focus => {
+                cx.on_next_frame(|_, cx| {
```

Comment:
These two is because the IME was being updated with the previous bound, leaving the IME panel at the previous focus.